x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -104,8 +104,7 @@ export const ACTION_AGENT_FILE_DOWNLOAD_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/{
 export const ACTION_STATE_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/state`;
 
 /** Endpoint Agent Routes */
-export const ENDPOINT_AGENT_STATUS_ROUTE = `/internal${BASE_ENDPOINT_ROUTE}/agent_status`;
-
+export const AGENT_STATUS_ROUTE = `${BASE_ENDPOINT_ROUTE}/agent_status`;
```

Comment:
Oh did we not want to expose it yet? I thought the idea of creating a service was so that we can expose it. Perhaps I misunderstood. 😓

Reply:
There is no need to make this a public API, so we had discussed during grooming that it would be internal. Serverless made us realize that unless really needed, we should stick with internal APIs unless there is a need for one to be public.

Reply:
changed it back to internal API 499f4e4e12f4f33455315a1942f94515284ed0ec